phoenix5-spark/README.md

```diff
@@ -89,7 +147,7 @@ public class PhoenixSparkRead {
 ### Save DataFrames to Phoenix using DataSourceV2
 
 The `save` is method on DataFrame allows passing in a data source type. You can use
-`phoenix` for DataSourceV2 and must also pass in a `table` and `zkUrl` parameter to
+`phoenix` for DataSourceV2 and must also pass in a `table` and `jdbcUrl` parameter to
```

Comment:
This is not true anymore (it was already wrong, but since you're updating the docs, let's fix this).
jdbcUrl is not mandatory.

Reply:
Done.